examples/tools/scene_viewer/scene_viewer_plugin.rs

```diff
@@ -44,9 +44,6 @@ Scene Controls:
     L           - animate light direction
     U           - toggle shadows
     C           - cycle through the camera controller and any cameras loaded from the scene
-    5/6         - decrease/increase shadow projection width
```

Comment:
could we keep some inputs for updating the cascade far and near settings?

Reply:
this is not blocking, could be a follow up

Reply:
Good idea. We could either allow control of just the number of cascades and the first and last far bounds and have the code calculate the bounds of each cascade, or allow full control. As it is meant to be illustrative, I'm inclined to do the former as it's a bit simpler until we have some UI slider stuff.

Reply:
sounds good, I will work on this as a follow up if that's ok with you